File: Res/Sorting/Insertion/insertion.py
def insertionSort(list):
    n = len(list)
    for i in range(1, n):
        key = list[i]
        j = i - 1
        while j >= 0 and list[j] > key:
            list[j + 1] = list[j]
            j = j - 1
        list[j + 1] = key
    return list
        
element = [78,59,62,41,52,75,12]
sortedList = insertionSort(element)
print(sortedList)

File: Res/Sorting/Insertion/insertion.dart
List<int> insertionSort(List<int> list) {
  int n = list.length;
  for (int i = 1; i < n; i++) {
    int key = list[i];
    int j = i - 1;
    while (j >= 0 && list[j] > key) {
      list[j + 1] = list[j];
      j = j - 1;
    }
    list[j + 1] = key;
  }
  return list;
}

void main() {
  List<int> list = [41, 45, 58, 96, 36, 52, 47];
  List<int> sortedList = insertionSort(list);
  print(sortedList);
}
